### PR TITLE
11117 bulk concept editor

### LINKED ISSUE #11117: Add bulk edit of concepts to the ETL Manager
The bulk data manager is a powerful feature in the Arches toolbox.  I would be nice to add the ability to reclassify concepts from one to another using this feature.

### PR DESCRIPTION
**Pull request by apeters**
### Types of changes

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Issues Solved

Closes #11117

### Checklist

-   I targeted one of these branches:
    - [x] dev/7.6.x (under development): features, bugfixes not covered below
    - [ ] dev/7.5.x (main support): regressions, crashing bugs, security issues, major bugs in new features
    - [ ] dev/6.2.x (extended support): major security issues, data loss issues
-   [ ] I added a changelog in arches/releases
-   [ ] I submitted a PR to arches-docs (if appropriate)
-   [ ] Unit tests pass locally with my changes
-   [ ] I added tests that prove my fix is effective or that my feature works
-   [ ] My test fails on the target branch

#### Accessibility Checklist

[Developer Guide](https://arches.readthedocs.io/en/stable/developing/advanced/accessibility/)

| Topic            | Changed | Retested |
| ---------------- | ------- | -------- |
| Color contrast   |         |          |
| Form fields      |         |          |
| Headings         |         |          |
| Links            |         |          |
| Keyboard         |         |          |
| Responsive Design|         |          |
| HTML validation  |         |          |
| Screen reader    |         |          |

#### Ticket Background
*   Sponsored by: Getty Digital Institute
*   Tested by: @apeters
*   Designed by: @vouidaskis, @apeters

### CODE REVIEW

**Review by apeters — COMMENTED** [review]
We should resolve the graph and node id to human readable names.
Screenshot 2024-08-08 at 2 27 43 PM

**Review by apeters — COMMENTED** [review]
this looks good to me and the functionality has been approved by Getty Digital

File: arches/app/templates/views/components/etl_modules/bulk_edit_concept.htm [hunk 1]
```diff
@@ -0,0 +1,280 @@
+{% extends "views/components/etl_modules/base-import.htm" %}
+{% load i18n %}
+
+{% block task_details %}
+<div class="etl-module-component-container">
+    <div class="etl-module-body">
+        <h2>
+            <span style="font-weight: 400; font-size: 1.7rem; color: #565656;">Replace Concept</span>
```
Comment (chiatt) [inline arches/app/templates/views/components/etl_modules/bulk_edit_concept.htm:8]: It looks like this and several other strings in this template could be in trans tags.

**Review by chiatt — CHANGES_REQUESTED** [review]

File: arches/app/templates/views/components/etl_modules/bulk_edit_concept.htm (hunk 1 above)
Comment (apeters) [reply arches/app/templates/views/components/etl_modules/bulk_edit_concept.htm:8]: Good catch!

Comment (apeters) [reply arches/app/templates/views/components/etl_modules/bulk_edit_concept.htm]: Ok, those items should be resolved @chiatt

**Review by chiatt — APPROVED** [review]

**Pull request merged**